src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,8 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        text = text.toLowerCase().replaceAll("\\W", "");
+        String txet = new StringBuffer(text).reverse().toString();
```

Comment:
weird name `txet`, why StringBuffer?

Reply:
StringBuffer() only for reverse() method